### ISSUE TITLE
TypeError: Reduce of empty array with no initial value

### ISSUE THREAD

**Issue by colindresj** [svg/svgo#400]
I'm using webpack and svgo-loader. Some svgs seem to result in this exception:

![out](https://cloud.githubusercontent.com/assets/4250234/8628055/90a699ae-271d-11e5-93bd-dfcc0141c9d8.png)

Here's what `cam.svg` looks like:

```xml
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg width="20px" height="17px" viewBox="0 0 20 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
    <!-- Generator: Sketch 3.3.2 (12043) - http://www.bohemiancoding.com/sketch -->
    <title>Rectangle 3 + Oval 1 Copy + Rectangle 7</title>
    <desc>Created with Sketch.</desc>
    <defs>
    </defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage">
        <g id="32" sketch:type="MSArtboardGroup" transform="translate(-6.000000, -8.000000)">
            <g id="Rectangle-3-+-Oval-1-Copy-+-Rectangle-7" sketch:type="MSLayerGroup" transform="translate(6.000000, 8.000000)">
                <path d="M20,3 L20,2 L1,2 L0,2 L0,16 L0,17 L19,17 L20,17 L20,3 Z M18,4 L2,4 L2,15 L18,15 L18,4 Z" id="Rectangle-3" fill="#252525" sketch:type="MSShapeGroup"></path>
                <g id="Oval-1-Copy">
                    <use stroke="#000000" stroke-width="2" sketch:type="MSShapeGroup" xlink:href="#path-1"></use>
                    <use stroke="none" xlink:href="#path-1"></use>
                </g>
                <rect id="Rectangle-7" fill="#252525" sketch:type="MSShapeGroup" x="0" y="0" width="6" height="2"></rect>
            </g>
        </g>
    </g>
</svg>
```

Any idea what might be causing this?

**Comment by GreLI** [comment]
Yeah, it's bug fixed in #394, it occurs when svgo optimize transform to empty state. In your case `translate(-6.000000, -8.000000)` and `translate(6.000000, 8.000000)` mutually cancel. As a workaround for the moment you can temporary disable `convertTransform`, then it should optimize normally.

**Comment by colindresj** [comment]
Cool thanks. I'll give that a shot for now
